src/components/App/App.jsx

```diff
@@ -65,12 +71,12 @@ class App extends React.Component {
 
   loadUser() {
     const github = new GitHub()
-    github.getCurrentUser().then(user => {
-      this.setState({ user })
-    }).catch(error => {
-      console.error('failed to load user', error)
-      GitHubAuth.deleteToken()
-    })
+    github.getCurrentUser()
+        .then(user => this.onUserLoad(user))
```

Comment:
I'd say either put the `.` on the preceding line and align `then` with `get`, or keep the `.` here and align it with `.get`.